drivers: eeprom: Fix at2x cs gpio flags macro typo

nandojve opened the pull request:
There is a typo on the macro that gets cs gpio flags. This fix the issue.

Signed-off-by: Gerson Fernando Budke <nandojve@gmail.com>

nandojve committed:
drivers: eeprom: Fix at2x cs gpio flags macro typo

There is a typo on the macro that gets cs gpio flags. This fix the
issue.

Signed-off-by: Gerson Fernando Budke <nandojve@gmail.com>

mbolivar-nordic reviewed (approved):
Looks like CI didn't catch this because the test_spi node in tests/drivers/build_all/app.overlay doesn't have any CS GPIOs.

Is that worth fixing too? Not a blocker.

nandojve commented:
> Looks like CI didn't catch this because the test_spi node in tests/drivers/build_all/app.overlay doesn't have any CS GPIOs.
> 
> Is that worth fixing too? Not a blocker.

Let me check!

nandojve committed:
tests: drivers: build_all: Fix bus address sequency

The test_spi_enc28j60 have a duplicated address at spi.dtsii file.
Add proper value for enc28j60 instance and fix address the sequency.

Signed-off-by: Gerson Fernando Budke <nandojve@gmail.com>

nandojve committed:
tests: drivers: build_all: Add cs-gpios entries for spi tests

The spi tests are not running because test_spi at app.overlay doesn't
have cs-gpios entry.  Add cs-gpios definition with one phandle for
every device defined at spi.dtsi.

Signed-off-by: Gerson Fernando Budke <nandojve@gmail.com>
